package.json

```diff
@@ -51,7 +51,9 @@
     "nock": "^13.0.0",
     "sinon": "^9.0.3",
     "snap-shot-it": "^7.0.0",
-    "typescript": "^3.8.3"
+    "typescript": "^3.8.3",
+    "js-yaml": "^4.0.0",
```

Comment:
I believe we need `js-yaml` to be an actual dependency, not a `dev-dependency` -- since you'll want it when running the application.

`@types/js-yaml` can stay as a dev-dependency.

Reply:
fixed